pkg/maps/lxcmap/lxcmap.go

```diff
@@ -197,9 +197,9 @@ func DeleteEntry(ip net.IP) error {
 // endpoint. It returns the number of errors encountered during deletion.
 func DeleteElement(f EndpointFrontend) []error {
 	errors := []error{}
-	for _, k := range f.GetBPFKeys() {
-		if err := LXCMap.Delete(k); err != nil {
-			errors = append(errors, fmt.Errorf("Unable to delete key %v in endpoint BPF map: %s", k, err))
+	for _, v := range f.GetBPFKeys() {
+		if err := LXCMap.Delete(v); err != nil {
+			errors = append(errors, fmt.Errorf("Unable to delete key %v in endpoint BPF map: %s", v, err))
```

Comment:
It might be worth keeping the `k` here, since these are the actual keys, and not values. It's already a little confusing getting keys and not values :/